reedoooo/cover-letter-frontend
# feat(user-settings): enhance UI and add icons to menu items

Enhanced the user settings menu by splitting it into left and right sections displayed side by side. Added icons to the beginning of each menu item for better user experience. Also applied a unique, professional font to the header of the Cover Letter Generator.

- Created styled components for LeftSection and RightSection to align content in columns
- Added Avatar and Logout button to the left section
- Placed menu items with icons (PersonIcon, DraftsIcon, SettingsIcon) in the right section
- Applied "Merriweather" font to the header of the Cover Letter Generator
- Ensured menu structure is responsive and accessible
- Adjusted icon sizes and styles for better visibility

TESTING:
To test these changes:
1. Open the user settings menu and verify the layout displays the avatar and logout button on the left, and menu items on the right.
2. Check that each menu item has an appropriate icon at the beginning.
3. Open the Cover Letter Generator and verify the header font is "Merriweather".

closes #123, #124
